### PR TITLE
1.11 - Updated volume mode field.

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-40228

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by anithamuthu79** [comment]
@qzhang  could you review the changes made in 1.11? If these changes look good to you, I will make the same change in previous releases. Ty.

File: pages/1.12/storage/persistent-volume/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Configure a persistent volume with the following options:
 ```
 
 - `containerPath`: The path where your application will read and write data. This must be a single-level path relative to the container; it cannot contain a forward slash (`/`). (`"data"`, but not `"/data"`, `"/var/data"` or `"var/data"`).
-- `mode`: The access mode of the volume. Currently, `"RW"` is the only possible value and will let your application read from and write to the volume.
+- `mode`: The access mode of the volume. Currently, `"RW"` and `"RO"` are the only possible values and will let your application read from and write to the volume.
```
Comment (qianzhangxa) [inline pages/1.12/storage/persistent-volume/index.md]: "RW" will let your application read from and write to the volume, and "RO" will only let your application read from the volume.

**Comment by qianzhangxa** [comment]
@anithamuthu79 Besides the changes about the volume mode field, I see you did some other changes which is not related to volume mode. Since the subject of this PR is `1.11 - Updated volume mode field.`, should those irrelevant changes be moved to a separate PR?

And I verified the `RW` and `RO` with Mesos but not with DC/OS, have you verified them in a DC/OS cluster from E2E? I guess we need to do that before updating the doc.

**Comment by anithamuthu79** [comment]
@qianzhangxa who is the engineer to verify your statement?
"And I verified the RW and RO with Mesos but not with DC/OS, have you verified them in a DC/OS cluster from E2E? I guess we need to do that before updating the doc."

File: pages/1.12/storage/persistent-volume/index.md [hunk 2]
```diff
@@ -451,7 +451,7 @@ The following example will create a pod with 2 containers and one shared persist
 
 ## Inspect and delete suspended stateful tasks
 
-To destroy and clean up persistent volumes and free the reserved resources associated with a task, perform 2 steps:
+Perform the following steps to free the reserved resources associated with a task, destroy and clean up persistent volumes.
```
Comment (qianzhangxa) [inline pages/1.12/storage/persistent-volume/index.md]: The right order is, destroy and clean up persistent volumes first, and then free the reserved resources associated with a task. So I think the original content is correct in this regard.

File: pages/1.12/storage/persistent-volume/index.md [hunk 3]
```diff
@@ -479,9 +479,8 @@ response:
 }
 ```
 
-**Note:** A running task will show `stagedAt`, `startedAt`, and `version` in addition to the information provided above.
-
-You can then
+**Note:** 
+A running task will show `stagedAt`, `startedAt`, and `version` in addition to the information provided above. You can do the following actions:
```
Comment (qianzhangxa) [inline pages/1.12/storage/persistent-volume/index.md]: I do not think `You can do the following actions:` should be part of the `Note`.

File: pages/1.12/storage/persistent-volume/index.md [hunk 4]
```diff
@@ -494,6 +493,6 @@ http DELETE http://dcos/service/marathon/v2/apps/postgres/tasks/postgres.53ab873
 
 After you have created your application, click the **Volumes** tab of the application detail view to get detailed information about your app instances and associated volumes.
 
-The Status column tells you if your app instance is attached to the volume or not. The app instance will read as "detached" if you have scaled down your application. Currently the only Operation Type available is read/write (RW).
+The `Status` column tells you if your app instance is attached to the volume or not. The app instance will read as "detached" if you have scaled down your application. Currently, the only operation type available are read/write (RW) and read only (RO). "RW" will let your application read from and write to the volume, and "RO" will only let your application read from the volume.
```
Comment (qianzhangxa) [inline pages/1.12/storage/persistent-volume/index.md]: `operation type` should be `operation types`?

**Comment by anithamuthu79** [comment]
@qianzhangxa could you review the changes on this PR?

**Comment by qianzhangxa** [comment]
LGTM!

**Comment by anithamuthu79** [comment]
@qianzhangxa can you approve this PR?

**Pull request closed without merging**